lib/src/message_grouper_state.dart

```diff
@@ -10,10 +10,10 @@ import 'package:protobuf/protobuf.dart';
 /// arrive.
 class MessageGrouperState {
   /// Reads the initial length.
-  _LengthReader _lengthReader;
+  late _LengthReader _lengthReader;
```

Comment:
How about we initialize this in place and remove the call to `reset` in the constructor body?

```suggestion
  var _lengthReader = _LengthReader();
```

Reply:
Done